thrift/cadence.thrift

```diff
@@ -94,6 +94,20 @@ service WorkflowService {
     )
 
   /**
+  * RestartWorkflowExecution restarts a previous workflow
+  * If the workflow is currently running it will terminate and restart
+  **/
+  shared.StartWorkflowExecutionResponse RestartWorkflowExecution(1: shared.RestartWorkflowExecutionRequest restartRequest)
+    throws (
+      1: shared.BadRequestError badRequestError,
+      3: shared.WorkflowExecutionAlreadyStartedError sessionAlreadyExistError,
```

Comment:
I think `2:` might be missing here. But overall, I think these are all 4XX errors, nice!

Reply:
woops will fix